app/models/concerns/orchestration.rb

```diff
@@ -151,6 +151,18 @@ def process(queue_name)
     fail_queue(q)
 
     rollback
+  ensure
+    unless q.nil?
+      logger.info("Processed #{q.completed.count}/#{q.all.count} orchestration tasks") if q.all.count > 0
```

Comment:
1. Add the queue_name here too, there are two.
1. `if q.all.count > 0` => `unless q.empty?`